pyomo/contrib/fbbt/interval.py

```diff
@@ -1,11 +1,18 @@
 import math
 import warnings
 import logging
-from pyomo.common.errors import DeveloperError, InfeasibleConstraintException
+from pyomo.common.errors import DeveloperError, InfeasibleConstraintException, PyomoException
 
 logger = logging.getLogger(__name__)
 
 
+class IntervalException(PyomoException):
+    pass
+
+
+feasibility_tol = 1e-8
```

Comment:
Why `feasibility_tol` here and `infeasibility_tol` in the other module?

Reply:
fixed